static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/useTransactionSamplesWebVitalsScoresQuery.tsx

```diff
@@ -121,7 +117,6 @@ export const useTransactionSamplesWebVitalsScoresQuery = ({
             'measurements.cls': toNumber(row['measurements.cls']),
             'measurements.ttfb': toNumber(row['measurements.ttfb']),
             'measurements.fid': toNumber(row['measurements.fid']),
-            'measurements.inp': toNumber(row['measurements.fid']),
```

Comment:
So this hook only queries for transactions/pageloads. Real INP data actually comes from spans, which is a different hook and dataset.

I'm just removing the pretend data completely because we won't be grabbing any INP measurements from here.